pkg/rpcclient/rpc.go

```diff
@@ -1285,3 +1285,27 @@ func (c *Client) TerminateSession(sessionID uuid.UUID) (bool, error) {
 
 	return resp, nil
 }
+
+// GetRawNotaryTransaction returns a fallback or main transaction that was
+// previously added to the memory pool by P2PNotaryRequest.
+func (c *Client) GetRawNotaryTransaction(hash util.Uint256) (*transaction.Transaction, error) {
+	var (
+		params = []any{hash.StringLE(), 1} // 1 for verbose.
```

Comment:
You need to implement this method exactly like `(c *Client) GetRawTransaction` and `(c *Client) GetRawTransactionVerbose`. Split it into two different methods.

Reply:
Done.